torch/csrc/stable/stableivalue_conversions.h

```diff
@@ -202,12 +202,21 @@ struct FromImpl<torch::headeronly::HeaderOnlyArrayRef<T>> {
       [[maybe_unused]] uint64_t extension_build_version,
       [[maybe_unused]] bool is_internal) {
     StableListHandle new_list_handle;
-    TORCH_ERROR_CODE_CHECK(
-        torch_new_list_reserve_size(val.size(), &new_list_handle));
-    for (const auto& elem : val) {
-      TORCH_ERROR_CODE_CHECK(torch_list_push_back(new_list_handle, from(elem)));
+    try {
+      TORCH_ERROR_CODE_CHECK(
+          torch_new_list_reserve_size(val.size(), &new_list_handle));
+      for (const auto& elem : val) {
+        TORCH_ERROR_CODE_CHECK(
+            torch_list_push_back(new_list_handle, from(elem)));
+      }
+      return from(new_list_handle);
+    } catch (const std::runtime_error& e) {
```

Comment:
don't you need to catch all flavors of exception? why is it ok to leak on things that aren't runtime_error?